src/aes_hash.rs

```diff
@@ -311,12 +315,15 @@ impl Hasher for AHasherFixed {
     }
 }
 
-#[cfg(feature = "specialize")]
-pub(crate) struct AHasherStr(pub AHasher);
+/// A specialized hasher for strings.
```

Comment:
This works for anything that can deref / cast to a `[u8]` so, String, str, slices of primitives etc.